studio/components/interfaces/Auth/Users/Users.tsx

```diff
@@ -109,7 +109,7 @@ const Users = () => {
         </div>
       </div>
       <section className="users-table-container mt-4 overflow-visible px-6">
-        <div className="section-block--body relative overflow-x-scroll rounded">
+        <div className="section-block--body relative overflow-x-auto rounded">
```

Comment:
i added this so that scrollbars will only show up when it starts to overflow

Reply:
aah nice!